src/Utilities/GameUtils.java

```diff
@@ -0,0 +1,90 @@
+package Utilities;
+
+import bricklets.Wall;
```

Comment:
tight coupling with the bricklets package is not so nice, maybe move the wall class to `entities/common/` ?